perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -48,6 +48,18 @@ sub geocoder_munge_results {
     }
 }
 
+sub new_report_title_field_label {
+    "Summarise the problem"
+}
+
+sub new_report_title_field_hint {
+    "Exact location, including any landmarks"
+}
+
+sub new_report_title_field_placeholder {
```

Comment:
This isn't a title field placeholder, this is the detail field `hint` (if you're calling the title one `hint` which is good, it's not actually an HTML placeholder). So I think every `new_report_title_field_placeholder` needs to be `new_report_detail_field_hint`.